app/page.tsx

```
@@ -107,63 +92,110 @@ export default function Home() {
-                  )}
-                </Card>
-              );
-            })}
+          <div className="grid grid-cols-3 gap-x-12 gap-24 mb-24">
+            <div>
+              <h1 className="text-3xl font-medium mb-4">
+                {content.cta.demo.title}
+              </h1>
+              <p
+                className="text-muted-foreground mb-8"
+                dangerouslySetInnerHTML={{ __html: content.cta.demo.text }}
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)